build/version.go

```diff
@@ -83,7 +83,7 @@ func VersionForType(nodeType NodeType) (Version, error) {
 
 // semver versions of the rpc api exposed
 var (
-	FullAPIVersion   = newVer(0, 14, 0)
+	FullAPIVersion   = newVer(0, 15, 0)
```

Comment:
@magik6k Do miner and worker versions need to be bumped? I bumped Full because of some new Msig stuff, I think?

Reply:
Nope, only full-node APIs got changes -> https://github.com/filecoin-project/lotus/compare/v0.7.0...master#diff-e1c890a5c832d59fa8c6615d3f41c70b